enterprise-suite/Makefile

```diff
@@ -43,7 +43,7 @@ script-tests: ## Run installer tests
 
 .PHONY: frontend-tests
 frontend-tests: ## Run the frontend tests against minikube
-	$(CHART_DIR)/scripts/run-e2e-tests.sh
+	$(CHART_DIR)/scripts/run-e2e-tests.sh all
```

Comment:
Don't need this change if you make running all tests the default.

Reply:
we still need to pass something,  because we have "set -u" in `run-e2e-tests.sh`
If we don't put any parameter, it will complain ```line 5: $1: unbound variable```